src/components/LeaderBoard.js

```diff
@@ -78,25 +78,21 @@ const useStyles = makeStyles()((theme) => ({
 
 function RibbonWrapper({ fill, index }) {
   const { classes } = useStyles();
-
   return (
     <div className={classes.ribbons}>
       <Ribbon fill={fill} />
-      <Typography variant="h5" className={classes.rank}>
-        {index + 1}
-      </Typography>
+      <figcaption className={classes.rank}>{index + 1}</figcaption>
```

Comment:
@martinyis thanks for the contribution!
We can stay on MUI and change the tag at the same time, check this doc:
https://mui.com/material-ui/api/typography/

Reply:
I agree with Dadior, we can use the component prop to change the element. The same goes for the other ones.